quarkus-workshop-super-heroes/super-heroes/rest-fights/src/test/java/io/quarkus/workshop/superheroes/fight/client/MockNarrationProxy.java

```diff
@@ -0,0 +1,22 @@
+package io.quarkus.workshop.superheroes.fight.client;
+
+import io.quarkus.test.Mock;
+import io.quarkus.workshop.superheroes.fight.Fight;
+import jakarta.enterprise.context.ApplicationScoped;
+import org.eclipse.microprofile.rest.client.inject.RestClient;
+
+@Mock
+@ApplicationScoped
+@RestClient
+public class MockNarrationProxy implements NarrationProxy {
+
+    @Override
+    public String narrate(Fight fight) {
+        return """
+            Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.
```

Comment:
Do we want a fallback that looks so obviously broken? Or should we have something 'acceptable but generic,' like "It was a mighty skirmish, but in the end there could be only one winner."

Reply:
Actually, I guess this is a mock, so it doesn't really matter what the content is. Although I'd maybe indicate it was mock-y in the text, to avoid people getting confused in the way I just did above. :)